cmd/kubestellar-where-resolver/cmd/where-resolver-cmd.go

```diff
@@ -83,39 +83,43 @@ func Run(ctx context.Context, options *resolveroptions.Options) error {
 	logger := klog.Background()
 	ctx = klog.NewContext(ctx, logger)
 
-	// create edgeSharedInformerFactory
-	espwRestConfig, err := options.EspwClientOpts.ToRESTConfig()
+	spaceManagementConfig, err := options.SpaceMgtOpts.ToRESTConfig()
 	if err != nil {
-		logger.Error(err, "failed to create config from flags")
+		logger.Error(err, "Failed to create space management config from flags")
 		return err
 	}
-	edgeClient, err := edgeclientset.NewForConfig(espwRestConfig)
+	spaceClient, err := spaceclient.NewMultiSpace(ctx, spaceManagementConfig)
 	if err != nil {
-		logger.Error(err, "failed to create clientset for KubeStellar Core Space")
+		logger.Error(err, "Failed to create space-aware client")
 		return err
 	}
-	edgeSharedInformerFactory := edgeinformers.NewSharedScopedInformerFactoryWithOptions(edgeClient, resyncPeriod)
+	spaceProviderNs := spacemanager.ProviderNS(options.SpaceProvider)
```

Comment:
I do not think the space provider (desc, right?) name should be a variable. It think it should be a static constant "default". The setup code insists on that, right? We do not intend to run with any other name for that SpaceProviderDesc, right?